infra/azure/terraform/network.tf

```diff
@@ -0,0 +1,38 @@
+# create virtual network
+resource "azurerm_virtual_network" "titus-vnet" {
+    name = "titus-vnet"
+    address_space = ["10.0.0.0/16"]
+    location = var.location
+    resource_group_name = var.resource_group_name
+} 
+
+resource "azurerm_subnet" "titus-subnet" {
+  name                 = "titus-subnet"
+  resource_group_name  = var.resource_group_name
+  virtual_network_name = azurerm_virtual_network.titus-vnet.name
+  address_prefixes     = ["10.0.1.0/24"]
```

Comment:
Drive the value of the network from Variables, allowing for better control.

Reply:
Done, vnet and subnet ranges defined as variables now